etl/steps/data/garden/ivs/2023-11-27/integrated_values_survey.meta.yml

```diff
@@ -3835,3 +3835,70 @@ tables:
         presentation:
           title_public: "A civil war: Average score"
 
+      happy:
+        title: "Happiness: Happy (aggregate)"
+        description_short: '% of respondents replying "very happy" or "quite happy" when asked "Taking all things together, would you say you are very happy, quite happy, not very happy or not at all happy?"'
+        display:
+          name: "Happiness: Happy"
+          <<: *common-display
```

Comment:
Woah what does '<<:' do here?

Reply:
[Mojmir taught me to do that](https://owid.slack.com/archives/C05KQ0LE05V/p1698159014402909?thread_ts=1698148615.354429&cid=C05KQ0LE05V). It is because if I only write 
```
        display:
          name: "Happiness: Happy"
```

the metadata for `display` is overridden, so no numDecimalPlaces, tolerance, etc. With <<: I can add both the name and the other common definitions.